docs/release-source/release/utils.md

```diff
@@ -30,7 +30,17 @@ Restores supplied method
 
 #### `sinon.createStubInstance(constructor);`
 
-Creates a new object with the given function as the protoype and stubs all implemented functions.
```

Comment:
Agreed. Having an explanation that says `function` and a code snippet with a `class` should be fine.